### PR TITLE
feat(MOPLAT-404): Markdown

### PR DESCRIPTION
**Pull request by rajsam003**
### Description

This PR resolves [MOPLAT-404]

https://user-images.githubusercontent.com/17421923/189368785-692491ed-6305-48e9-9b11-cbc4dc8d80d6.mov

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-404]: https://artsyproduct.atlassian.net/browse/MOPLAT-404?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by patrinoua** [comment]
Cool Sam! 🚀
Is there a way to use our design elements when displaying Markdown? Meaning the same text font, way to underline etc. It looks a bit funky currently.

File: src/app/sharedUI/molecules/Markdown.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+const basicRules = defaultRules({})
+
+function stringifyChildren(children: string | string[]): string {
+  return _.isArray(children) ? children.join("") : children
+}
+
+export const Markdown = ({ rules = basicRules, children, ...rest }: Props & FlexProps) => {
+  return <Flex {...rest}>{renderMarkdown(stringifyChildren(children ?? ""), rules)}</Flex>
```
Comment (patrinoua) [inline src/app/sharedUI/molecules/Markdown.tsx:17]: maybe if we add a <Text> here?

Comment (pvinis) [reply src/app/sharedUI/molecules/Markdown.tsx:17]: no, this is not needed. if you look at https://github.com/artsy/energy/pull/184/files#diff-b032863d80ba288b3b6c8938588512d4bc8f5c5e304fdfc08502f47e57b2d2dfR51, the `renderMarkdown` is the function that will return a `<Text>` if its needed. sometimes it returns other things.

**Review by pvinis — APPROVED** [review]

File: src/app/sharedUI/molecules/Markdown.tsx (hunk 1 above)
Comment (patrinoua) [reply src/app/sharedUI/molecules/Markdown.tsx:17]: Sure but this is Text imported from 'react-native' and not 'palette', so it does not matching our design.

Screen Shot 2022-09-13 at 11 22 54

**Pull request merged**